js/src/ui/AccountCard/accountCard.js

```diff
@@ -88,6 +93,7 @@ export default class AccountCard extends Component {
             className={ styles.balance }
             showOnlyEth
           />
+          { (password) ? this.renderPassword() : null }
```

Comment:
@ngotchac Happy with the approach or would passing children from the renderer be more appropriate? (Not unhappy, just concerned that this may blow up as we add more stuff - and currently the password is a single-instance use as well)